### PR TITLE
Add userd ID for Gateway

### PR DESCRIPTION
**Pull request by pipo02mix**
Towards https://github.com/giantswarm/thg/issues/114
This PR:

- adds/changes/removes etc

### Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by webwurst** [comment]
Looks good. Is this something from upstream? Should we contribute else there?

File: helm/linkerd2-multicluster-target-app/values.yaml [hunk 1]
```diff
@@ -54,3 +54,6 @@ enablePSP: true
 # across hosts and zones for High Availability.
 # Enable this only when you have multiple replicas of components.
 enablePodAntiAffinity: true
+
+# -- User ID for the gateway
+controllerUID: 2103
```
Comment (ubergesundheit) [inline helm/linkerd2-multicluster-target-app/values.yaml:59]: Please also add this key to the [values schema](https://github.com/giantswarm/linkerd2-multicluster-target-app/blob/main/helm/linkerd2-multicluster-target-app/values.schema.json)

**Review by ubergesundheit — CHANGES_REQUESTED** [review]
LGTM with changed values schema

File: helm/linkerd2-multicluster-target-app/values.yaml (hunk 1 above)
Comment (pipo02mix) [reply helm/linkerd2-multicluster-target-app/values.yaml:59]: done

**Comment by pipo02mix** [comment]
This is done in other deployments upstream but not in gateway (I think it is missed) I will push there too

**Review by ubergesundheit — APPROVED** [review]

**Pull request merged**